src/core/compact_object.cc

```diff
@@ -556,6 +556,20 @@ void RobjWrapper::SetString(string_view s, MemoryResource* mr) {
   }
 }
 
+void RobjWrapper::ReserveString(size_t size, MemoryResource* mr) {
```

Comment:
This gets called once per big string right ? So if for example I have a 600kb string, I will read it in 3 chunks and ReserveString will only be called once right ?

Reply:
true